courses/templates/partials/course-carousel.html

```diff
@@ -1,5 +1,5 @@
 {% load static wagtailimages_tags %}
-<div class="course-in-program">
+<div id="courseLineup" class="course-in-program">
```

Comment:
Camel case is the convention on html IDs. Doesn't make a difference but since we will be using these most in JS, makes sense to stick with JS conventions.

Reply:
Not sure though, fragment IDs are shown in urls as well 😕

Reply:
That's standard as far as link to an element on an html page is concerned.